Drivers/Inc/CAN.h

```diff
@@ -29,21 +36,25 @@ typedef struct {
  * @param   None
  * @return  None
  */
-void CANbus_Init(void);
+void CAN_Init(void);
 
 /**
  * @brief   Transmits data onto the CANbus
  * @param   id : CAN id of the message
  * @param   payload : the data that will be sent.
  */
-int CANbus_Send(CANId_t id, CANPayload_t payload);
+int CAN_Send(CANId_t id, CANPayload_t payload);
 
 /**
  * @brief   Checks if the CAN ID matches with expected ID and then copies message to given buffer array
  * @param   CAN line bus
  * @param   pointer to buffer array to store message
  * @return  0 if ID matches and 1 if it doesn't
  */
+<<<<<<< Updated upstream
 int CANbus_Read(CAN_t bus, uint8_t* buffer);
+=======
+int CAN_Read(CAN_t bus, uint32_t id, uint8_t* buffer);
+>>>>>>> Stashed changes

```

Comment:
Fix this merge conflict

Reply:
What is a merge conflict? Like something is wrong in the code?

Reply:
The whole <<<< and >>>> stuff. Means there are 2 versions of that file tracked by Git and it just combined them. You have to essentially choose the right one and delete the other one.

Reply:
I think it's good now.